Repo-Integration/Encryption/README.md

```diff
@@ -0,0 +1,39 @@
+## Encrypting Private Registry Credentials for hostRule Configurations
+
+Mend Repository Integration customers often require encrypted credentials to access private registries for package resolution.
+
+Typically, users can encrypt credentials through the following methods:
+
+- **Cloud-Hosted Integrations:** [Mend Repo Integration Encryption page](https://mend-resources.mend.io/index-gh.html)
```

Comment:
Add link to Github.com, ADO, and Bitbucket below here.  Got you started with GH.com
https://docs.mend.io/bundle/integrations/page/configure_mend_for_github_com_to_resolve_your_private_dependencies.html